api/nucleus/views.py

```diff
@@ -1,16 +1,22 @@
-from django.shortcuts import render
+from django.conf import settings
+from django.shortcuts import redirect, render
 from social_django.utils import psa, load_strategy
 
 
 def require_email(request):
```

Comment:
This view redirects to the signup complete page after signing up as a new user via steam so that we can collect additional info (email address etc.).